src/daemon/message_hexdump_test.go

```diff
@@ -0,0 +1,154 @@
+package daemon
+
+import (
+	"testing"
+
+	"fmt"
+
+	"github.com/skycoin/skycoin/src/cipher"
+	"github.com/skycoin/skycoin/src/coin"
+	"github.com/skycoin/skycoin/src/daemon/pex"
+	"github.com/skycoin/skycoin/src/testutil"
+)
+
+func TestIntroductionMessage(t *testing.T) {
```

Comment:
These methods should start with `Example` not `Test`

Reply:
https://blog.golang.org/examples

Reply:
Signature for testutil.RandSHA256() includes t *testing.T
Should I change to my local GenerateRandomSHA256()?

Reply:
@stdevEdu I'll take this from here since I'm gonna need this asap for another task . Please focus on the other open issues assigned to you .